frontend/src/views/Join.tsx

```diff
@@ -67,23 +103,25 @@ function JoinGamePage() {
                 event.preventDefault();
               }
               if (event.key === 'Enter' && validRoomId) {
```

Comment:
Add the `loading` parameter here as well, to prevent spamming of pressing that key.

Reply:
Sounds good - I also added a check in the if statement above to prevent default if the user has already entered 6 digits, lmk if you want me to undo that but I thought that it could be useful.